### ISSUE TITLE
Query builder features from angular-version missing in react-version

### ISSUE THREAD

**Issue by tig987** [grafana/grafana#61186]
**What happened**:
After an upgrade from Grafana 9.2 to 9.3,2 the query builder in a panel does not show the advanced UI for TimescaleDB queries anymore but only the standard UI for normal Postgresql queries

**What you expected to happen**:
the advanced UI for TimescaleDB queries should still be available

**How to reproduce it (as minimally and precisely as possible)**:
upgrade from Grafana 9.2 to 9.3,2

**Anything else we need to know?**:

**Environment**:
- Grafana version: 9.3.2
- Data source type & version: Postgesql 13 + TimescaleDB 2.9.1
- OS Grafana is installed on: Debian 11
- User OS & Browser: Windows 10, Vivaldi 5.6
- Grafana plugins: none
- Others:

**Comment by usmangt** [comment]
Hi @tig987,

Thanks for opening this issue.

Was the query builder option available in the previous version (9.2.x)?

Can you share the screenshot if possible?

**Comment by djzort** [comment]
I can confirm this on the same set up as the above, but alas i have no screenshot

**Comment by Kreutzweise** [comment]
Hi guys,

i got the same problem unfortunately. In the past it looked like that 
![image](https://user-images.githubusercontent.com/96666303/212557008-1a60bea9-ee7a-47de-941f-08565d5567b3.png)

Now the builder is only 
![image](https://user-images.githubusercontent.com/96666303/212557017-ede8fbd3-dc64-4d75-869e-c2d408a1ef0b.png)

Do you have any idea why it might have changed?

**Comment by usmangt** [comment]
Hi all,

I will forward this issue to our Engineers as they might be able to tell in more detail.

**Comment by natellium** [comment]
I believe this was intentionally introduced by https://github.com/grafana/grafana/pull/52831. cc @grafana/grafana-bi-squad

**Comment by zoltanbedi** [comment]
Hey everyone,
Let me help you with this issue. 
@tig987 What is your issue exactly? What I understand and what you might face is that you had a query that you made in the query builder but you see the code editor now instead. This is because we migrated the SQL datasources to React so the query editor was rebuilt and the new one is not compatible with the old one so we can only show the raw query in the code editor. You need to recreate your query in the new query builder if you prefer that.

@Kreutzweise does the above make sense to you? Did it answer your question?

**Comment by tig987** [comment]
It is not possible to recreate the query in the new query builder
because the new query builder does not offer the options necessary to build a TimescaleDB SQL query.

**Comment by zoltanbedi** [comment]
> It is not possible to recreate the query in the new query builder
> because the new query builder does not offer the options necessary to build a TimescaleDB SQL query. 

Thanks for getting back to me @tig987. Can you please give me an example of query that is not possible to create?

**Comment by Kreutzweise** [comment]
@zoltanbedi thanks for the explaination! I was a die hard Influxdb 1.x user and need/want to change the DB. In my process of evaluating the options I came across this issue and was not sure if I'm doing something wrong. 
The advanced query builder was a neat feature for people who don't have the knowledge for SQL and therefore a big plus in my considerations. I also happen to struggle to work with the "old query builder" for example I couldn't find out where to set the Macros such as $__timeFilter. Is there a documentation on the possibilities of the current query builder? Asking for a noob ;) 

Thanks for the effort and amazing work btw.!

> Hey everyone, Let me help you with this issue. @tig987 What is your issue exactly? What I understand and what you might face is that you had a query that you made in the query builder but you see the code editor now instead. This is because we migrated the SQL datasources to React so the query editor was rebuilt and the new one is not compatible with the old one so we can only show the raw query in the code editor. You need to recreate your query in the new query builder if you prefer that.
> 
> @Kreutzweise does the above make sense to you? Did it answer your question?

**Comment by zoltanbedi** [comment]
@tig987 we will add back the time and metric column selector which will produce the same query as before. I took a look and now see what you meant.
In this screenshot I have a postgresql query and we should make sure to be able to produce the same query in the new query builder.
![Screenshot 2023-01-24 at 15 27 06](https://user-images.githubusercontent.com/13729989/214325853-55c24542-918a-4959-bef5-8f46b6c4be9b.png)

When timescaledb is enabled and there is a time column selected we produce a different query.
![Screenshot 2023-01-24 at 15 33 41](https://user-images.githubusercontent.com/13729989/214326498-d4a81024-55fc-4c1c-8df4-35f20fb6653e.png)

@Kreutzweise You can find the doc for the query builder here https://grafana.com/docs/grafana/latest/datasources/postgres/#query-builder . I know it is not much but we are open for feedback and suggestions. Right now you can't set macros in the query builder UI. Our thinking was that the query builder should be used by beginners and folks who want to query their db quick and for any other advanced query there is the code editor. Although I think we will add the time and metric column handling and with this the macro support the the query builder.

**Comment by Penagwin** [comment]
I'm using just postgres without timescale - but I'm missing a critical time based feature.

The old UI used to have that "Time column" that added `$__timeGroupAlias` and `$__timeFilter` to the query.

As far as I can tell this is not possible with the current UI. Without that time column AFAIK I'm unable to build a simple query that is essentially "How many orders were there, grouped by each hour" in the Query Builder, which I think has been a feature for like 5 years?

There's a lot of queries that require those macros and used to be very easy to build.

**Comment by zoltanbedi** [comment]
This is the same as #68750

**Comment by thalesmaoa** [comment]
Not sure if here it the proper place, but there is another problem with the new query builder. I can't manually fill the tables names.

Previously, query builder were able to find tables in public schemas. Just drop down and they were all there. 
Besides, if I needed to access tables inside other schemas, I just wrote schema.table, it was able to find. 

In the new query builder it doesn't happen. It doesn't accept manual schema.table fill.
